tests/Unit/Utilities/Test_StdHelpers.cpp

```diff
@@ -147,3 +148,22 @@ SPECTRE_TEST_CASE("Unit.Utilities.StdHelpers.StdArrayArithmetic",
     CHECK(gsl::at(neg_p1, i) == approx(gsl::at(expected_neg_p1, i)));
   }
 }
+
+SPECTRE_TEST_CASE("Unit.Utilities.StdHelpers.StdArrayMagnitude",
+                  "[DataStructures][Unit]") {
+  const std::array<double, 2> p1{{3., 4.}};
+  CHECK(magnitude(p1) == approx(5.));
+  const std::array<double, 3> p2{{
```

Comment:
Can this be on one line?

Reply:
Interestingly this was what clang-format was doing because of the stray comma at the end of 157, I didn't realise that would even compile.